app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -6,6 +6,12 @@ import PreloadStore from "discourse/lib/preload-store";
 import Category from "discourse/models/category";
 import User from "discourse/models/user";
 
+const topicTrackingFilterFns = [];
```

Comment:
make sure we don't pile some state between tests

Reply:
how?